src/utils.py

```diff
@@ -288,3 +281,13 @@ def ymdhms(dt):
     if dt:
         dt = todt(dt) # convert to utc, etc
         return rfc3339(dt, utc=True)
+
+def sortdict(d):
+    "imposes alphabetical ordering on a dictionary. returns an OrderedDict"
+    if isinstance(d, list):
```

Comment:
I just had a thought @gnott , that if a `OrderedDict` was encountered, then I could consider the results already ordered and avoid the forced ordering, preserving the ordering efforts you're doing in elife-tools, but it looks like some of the `OrderedDict` results contain unordered nested items. Looks mostly like `additionalFiles` and `figure` elements.

Reply:
sorry, the context is that it's this function sorting everything that is returned from `elife-tools`. I was thinking if it were asked to order an ordereddict I could reasonably assume it was already ordered and return it as-is, but elife-tools is returning OrderedDicts whose contents contain unordered items. Sorry to be confusing.

Reply:
Is there a test for ``sortdict()`` that has an ``OrderedDict()`` as the input, and does it sort it or let it pass through in its default order? I guess that's the precise question I was wondering about.

And thanks for clarifying this would sort everything returned from the parser functions.

Reply:
there are tests: https://github.com/elifesciences/bot-lax-adaptor/pull/377/files#diff-3801658a7002f05f853a58bfb363ab79R94

but not for that specific case. OrderedDicts are sorted. The logic is very simple.

Reply:
``OrderedDict`` sort order would be in the order items are added, and not alphabetical (I believe, I read that again yesterday during some refactoring). If that is sufficient, which it would be for making test assertions, is all cool!  :+1: